assets/src/components/shortcuts.js

```diff
@@ -23,13 +25,26 @@ const Shortcuts = ( { insertBlock } ) => {
 					onClick={ () => insertBlock( block ) }
 					label={ blockType.title }
 					labelPosition="bottom"
+					disabled={ ! canInsertBlockType( block ) }
```

Comment:
Should the CTA button even be displayed when on the first page? It seems strange to have a button appearing by default which is disabled.